yeahcheese/resources/js/components/EventEditorComponent.vue

```diff
@@ -2,101 +2,182 @@
+          >
+
+          <div
+            v-if="!validateStatus.title"
+            class="alert alert-danger my-2"
+            role="alert"
+          >
+            {{ validateErrorMessages.title }}
+          </div>
+        </div>
+
+        <div class="form-group">
+          <label>公開開始日</label>
+          <input
+            v-model="release_date"
+            type="date"
+            class="form-control"
+          >
+
+          <div
+            v-if="!validateStatus.releaseDate"
+            class="alert alert-danger my-2"
+            role="alert"
+          >
+            {{ validateErrorMessages.releaseDate }}
+          </div>
+        </div>
+
+        <div class="form-group">
+          <label>公開終了日</label>
+          <input
+            v-model="end_date"
+            type="date"
+            class="form-control"
+          >
+
+          <div
+            v-if="!validateStatus.endDate"
+            class="alert alert-danger my-2"
+            role="alert"
+          >
+            {{ validateErrorMessages.endDate }}
+          </div>
+        </div>
       </div>
 
-      <button type="submit" class="col-sm-2 btn btn-primary" @click="updateEvent">更新</button>
+      <button
+        type="submit"
+        class="col-sm-2 btn btn-primary"
+        @click="updateEvent"
+      >
+        更新
+      </button>
     </div>
   </div>
 </template>
 
 <script>
 import api from '../api';
-import Vue from 'vue';
 
 export default {
   name: 'EventEditorComponent',
   props: {
-    eventId: Number,
+    eventId: {
+      type: Number,
+      require: true,
+      'default': -1,
+    },
+    validateErrorMessages: {
+      type: Object,
+      require: false,
+      'default': () => ({
+        title: "1文字以上255文字以下のタイトルを入力してください。",
+        releaseDate: "公開開始日は公開終了日以前にしてください。",
+        endDate: "公開終了日は公開開始日以降にしてください。",
+      }),
+    },
+    postStatusMessages: {
+      type: Object,
+      require: false,
+      'default': () => ({
+        success: "イベント情報が更新されました",
+        failed: "更新できませんでした",
+      }),
+    },
   },
   data: function () {
     return {
       title : '',
       release_date : '',
       end_date : '',
-      message : '',
-      error_title_msg : '',
-      error_release_date_msg : '',
-      error_end_date_msg : '',
-      error_flag : false,
+      errorMessages: [],
+      postStatusMessage : '',
+      validateStatus: {
+        title: true,
+        releaseDate: true,
+        endDate: true,
+      }
     }
   },
-  created: function () {
-    api.fetchEvent(this.eventId).then(
-      eventResponse => {
-        const data = eventResponse.data.data;
-        this.title = data.title;
-        this.release_date = data.release_date;
-        this.end_date = data.end_date;
-      },
-      // TODO: API利用に失敗した際の処理を記述する リクエストの再送信など
-      errors => console.error(errors)
-    );
+  computed: {
+    isUpdateSuccess () {
+      return (this.postStatusMessage == this.postStatusMessages.success ? true : false);
```

Comment:
- 括弧いらなそう
- 基本的に厳密比較をした方が型安全
- `this.postStatusMessage` と `this.postStatusMessages.success` が同じかどうか見ればいいから、` ? true : false` はいらない

```suggestion
      return this.postStatusMessage === this.postStatusMessages.success;
```

Reply:
ご指摘頂いた部分を修正しました